### ISSUE TITLE
Failing to understand encoding of inequalities

### ISSUE THREAD

**Issue by rolfmorel** [potassco/clingoLP#18]
I am having trouble understanding how linear inequalities should be encoded.

Consider `test1.pl`:
```
&sum { a } = 1.
&sum { -1*a } = 1.
```
Running `python -m clingolp.app test1.pl --show-lp-solution` gives me output:
```
ClingoLP version 0.2.0
Reading from test.pl
Solving...
Answer: 1

LP solution: optimum=0.0
a=1.0 ((-1)*a)=0.0 
LP solver calls: 1   Time lp_solve :  5.2634000000023606e-05

SATISFIABLE
```

Program `test2.pl`:
```
&sum { a ; -1*a } = 1.
```
gives me the exact same assignment.

I was expecting `-1*a` to be treated as "variable `a` with coefficient `-1`". From the above output, it appears as though this multiplication (`-1*a`) is treated as a term (`((-1)*a)`) that is its own variable, with no relationship to variable `a`.

Consider program `test3.pl`:
```
&sum { b } >= 0.
&sum { a } >= 2.
&sum { a ; -1*b } <= 1.
```
where I mean to express:
```
b >= 0
a >= 2
a - b <= 1
```
Running `python -m clingolp.app test3.pl` returns `UNSATISFIABLE`.

Only `test4.pl`:
```
&sum { a } = 1.
&sum { a } = 0.
```
works as expected, in that it is UNSAT.

Could you please explain how one should go about properly encoding inequalities?

**Comment by sthiele** [comment]
I think that might be a bug related to clingo 5.5.0. Can you please send the result of `clingoLP --version`.

**Comment by rolfmorel** [comment]
Here you go:
```
$ python -m clingolp.app --version
clingoLP version 0.2.0
Address model: 64-bit

libclingo version 5.5.0
Configuration: without Python, without Lua

libclasp version 3.3.6 (libpotassco version 1.1.0)
Configuration: WITH_THREADS=1
Copyright (C) Benjamin Kaufmann

License: The MIT License <https://opensource.org/licenses/MIT>
```

**Comment by sthiele** [comment]
As a work around you may try with the old clingoLP version 0.1.0.
`conda install -c potassco -c conda-forge clingo-lp clingo=5.4`

**Comment by sthiele** [comment]
This should be fixed in PR #19

**Comment by rolfmorel** [comment]
Thanks Sven, that fixed my issues!
